common/src/api/external/mod.rs

```diff
@@ -616,6 +616,7 @@ pub enum ResourceType {
     RoleBuiltin,
     UpdateAvailableArtifact,
     UserBuiltin,
+    VirtualProvision,
```

Comment:
I couldn't see where this was used.  What resource or object does it actually describe?

Reply:
It was used in an error, but I can remove that: fc81adaed1910ec006c7eee9a9b27339ba4a90d0